src/app/feed/feed-create-post/feed-create-post.component.ts

```diff
@@ -234,8 +244,64 @@ export class FeedCreatePostComponent implements OnInit {
   }
 
   _handleFileInput(file: File) {
-    if (!file.type || !file.type.startsWith("image/")) {
-      this.globalVars._alertError("File selected does not have an image file type.");
+    if (!file) {
+      return;
+    }
+
+    if (!file.type || (!file.type.startsWith("image/") && !file.type.startsWith("video/"))) {
+      this.globalVars._alertError("File selected does not have an image or video file type.");
+      return;
+    }
+    if (file.type.startsWith("video/")) {
+      // if (file.size > 200 * (1024 * 1024)) {
+      //   this.globalVars._alertError("File is too large. Please choose a file less than 200MB");
+      //   return;
+      // }
+      let upload: tus.Upload;
+      let mediaId = "";
+      const comp: FeedCreatePostComponent = this;
+      const options = {
+        endpoint: this.backendApi._makeRequestURL(environment.uploadVideoHostname, BackendRoutes.RoutePathUploadVideo),
+        chunkSize: 50 * 1024 * 1024, // Required a minimum chunk size of 5MB, here we use 50MB.
+        // resume: true,
```

Comment:
Do we need this?

Reply:
no I'll specify what other options are available and where to look for the documentation on them.